src/index.ts

```diff
@@ -0,0 +1,72 @@
+/* Observable */
+export { Observable } from './Observable';
+export { ConnectableObservable } from './observable/ConnectableObservable';
+export { Operator } from './Operator';
+export { observable } from './symbol/observable';
+
+/* Subjects */
+export { Subject } from './Subject';
+export { BehaviorSubject } from './BehaviorSubject';
+export { ReplaySubject } from './ReplaySubject';
+export { AsyncSubject } from './AsyncSubject';
+
+/* Schedulers */
+export { asap as asapScheduler } from './scheduler/asap';
+export { async as asyncScheduler } from './scheduler/async';
+export { queue as queueScheduler } from './scheduler/queue';
+export { animationFrame as animationFrameScheduler } from './scheduler/animationFrame';
+export { VirtualTimeScheduler, VirtualAction } from './scheduler/VirtualTimeScheduler';
+
+/* Subscription */
+export { Subscription } from './Subscription';
+export { Subscriber } from './Subscriber';
+
+/* Notification */
+export { Notification } from './Notification';
+
+/* Utils */
+export { pipe } from './util/pipe';
+export { noop } from './util/noop';
+export { identity } from './util/identity';
+
+/* Error types */
+export { ArgumentOutOfRangeError } from './util/ArgumentOutOfRangeError';
+export { EmptyError } from './util/EmptyError';
+export { ObjectUnsubscribedError } from './util/ObjectUnsubscribedError';
+export { UnsubscriptionError } from './util/UnsubscriptionError';
+export { TimeoutError } from './util/TimeoutError';
+
+/* Static observable creation exports */
+export { bindCallback } from './observable/bindCallback';
+export { bindNodeCallback } from './observable/bindNodeCallback';
+export { combineLatest } from './observable/combineLatest';
+export { concat } from './observable/concat';
+export { defer } from './observable/defer';
+export { empty } from './observable/empty';
+export { forkJoin } from './observable/forkJoin';
+export { from } from './observable/from';
+export { fromEvent } from './observable/fromEvent';
+export { fromEventPattern } from './observable/fromEventPattern';
+export { generate } from './observable/generate';
+export { _if as iif } from './observable/if';
+export { interval } from './observable/interval';
+export { merge } from './observable/merge';
+export { of } from './observable/of';
+export { onErrorResumeNext } from './observable/onErrorResumeNext';
+export { pairs } from './observable/pairs';
+export { race } from './observable/race';
+export { range } from './observable/range';
+export { _throw as throwError } from './observable/throw';
+export { timer } from './observable/timer';
+export { using } from './observable/using';
+export { zip } from './observable/zip';
+
+/* Constants */
+export { EMPTY } from './observable/empty';
+export { NEVER } from './observable/never';
+
+/* Types */
+export * from './types';
+
+/* Config */
+// export { config } from './config';
```

Comment:
?

Reply:
Config didn't exist in stable/5.5... that was new from 6.0.